agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml

```diff
@@ -61,3 +61,11 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - extensions
+  resourceNames:
+  - sysdig-agent
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
```

Comment:
The podsecuritypolicy settings seem reasonable to me. but to ensure that the policy is consciously enabled, can  the "verbs: use" setting be commented out?

Reply:
Nope. I don't believe PSP works without that